docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
   postgres:
```

Comment:
Nice job, Isaac! You could probably just do a find-replace all to delete the occurrences of `, defaults to None` everywhere that parameters don't actually have a `param: Optional[type] = None`. But yeah, if it all works, once you've removed those descriptions from the docstrings, let's get this merged!

Next step is deploying it to AWS :D